Makefile

```diff
@@ -261,6 +261,8 @@ endef
 define generate-operator-manifests
 	$(call generate-manifests-from-helm,scylla-operator,helm/scylla-operator,$(1),$(3))
 
+    $(YQ) eval -i -P '.spec.template.spec.containers[0].image = "$(IMAGE_REF)"' '$(3)'/scylla-operator/templates/deployment.yaml
```

Comment:
wouldn't we rather change the default in the helm template and this would get it for free?